libs/sdk-ui-dashboard/src/presentation/scheduledEmail/types.ts

```diff
@@ -51,6 +51,11 @@ export interface IScheduledEmailManagementDialogProps {
      */
     onAdd?: () => void;
 
+    /**
+     * Callback to be called, when scheduled email is deleted.
+     */
+    onDelete?: () => void;
```

Comment:
Shouldn't the argument type be mentioned here?

Reply:
Renaming the prop so that it makes more sense